apps/dashboard/src/components/auth/create-organization.tsx

```diff
@@ -60,7 +61,41 @@ function FormContainer({ children }: FormContainerProps) {
 }
 
 function OrganizationForm() {
-  return <OrganizationListForm appearance={FORM_APPEARANCE} {...ORGANIZATION_FORM_CONFIG} />;
+  const [showRegionSelector, setShowRegionSelector] = useState(false);
+
+  useEffect(() => {
+    // Watch for DOM changes to detect when we're on the form page (Page 2)
+    const observer = new MutationObserver(() => {
+      // Check if the organization creation form (with name input) is visible
+      const nameInput = document.querySelector('input[name="name"]');
+      const isOnFormPage = !!nameInput;
+
+      if (isOnFormPage !== showRegionSelector) {
+        setShowRegionSelector(isOnFormPage);
+      }
+    });
+
+    // Start observing
+    observer.observe(document.body, {
+      childList: true,
+      subtree: true,
+    });
+
+    return () => observer.disconnect();
+  }, [showRegionSelector]);
+
+  return (
+    <div className="relative">
+      {/* Region selector - only visible on Page 2 (form page), aligned with form content */}
+      {showRegionSelector && (
+        <div className="absolute -top-14 left-4 z-20">
+          <RegionSelector />
+        </div>
```

Comment:
Will this be visible to all users? Can we control this to specific users?

Reply:
Its controlled via a LD flag in Selector component. I tested this with LaunchDarkly. Do you want me to bring it outside?
https://github.com/novuhq/novu/blob/f1d5b038117b804fb026e8d2ed9e35223fff0903/apps/dashboard/src/context/region/region-selector.tsx#L18-L19